config/metrics.js

```diff
@@ -113,6 +113,10 @@ module.exports = {
 		extract: (item) => !!getBPRules(item)?.allowsDeletions,
 		permissions: ['ADMIN'],
 	},
+	AllowsSecurity: {
+		extract: (item) => !!getBPRules(item)?.allowsSecurity,
+		permissions: ['ADMIN','WRITE'],
+	},
```

Comment:
@ljharb  well i believe that all the commits should be removed and let the last one only, so i will remove this lines of code because the metric was already there but we needed to reverse it. just give me a minute